pytest_sphinx.py

```diff
@@ -16,10 +16,15 @@
 import sys
 import traceback
 
+from pkg_resources import parse_version
```

Comment:
Note that this adds quite some overhead.
See https://github.com/pytest-dev/pytest-django/pull/826

Reply:
As for the change itself you could also just look for the `from_parent` attribute, or require pytest 5.4 (like pytest-asyncio does it).  I'd prefer the former.

Reply:
Thx @blueyed for this info! I've just fixed this in the master branch/